spec/models/course/notification_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Course::Notification, type: :model do
```

Comment:
so what is the difference between a unit test and integration test? What should you be testing for each of them? Do they duplicate?